src/main/resources/application.yaml

```diff
@@ -30,3 +30,6 @@ service:
       sas-validity: 300 #In seconds
     - name: sscs
       sas-validity: 300 #In seconds
+
+flyway:
+  skip-migrations: ${FLYWAY_SKIP_MIGRATIONS:false}
```

Comment:
best to avoid having default values as it makes it easy if containers don't come and fail

Reply:
can we please remove default value ?

Reply:
kk